docs/test-api-matrix.md

```diff
@@ -5,7 +5,7 @@ Endpoint | binding functions | basic CRUD test | deeper test| notes/component/st
 **Application Inventory**||||
 application|:white_check_mark: partially|:heavy_check_mark:||
 bucket||||partially within application
-dependency||||
+dependency|:white_check_mark:|:white_check_mark:|:white_check_mark:|
```

Comment:
Please use ```:heavy_check_mark:``` since with CRUD & cyclic check this test Dependency should be fully covered.